src/FSharp.Control.TaskSeq/TaskSeqInternal.fs

```diff
@@ -613,6 +631,103 @@ module internal TaskSeqInternal =
                     | false -> ()
         }
 
+
+    let skipOrTake skipOrTake count (source: TaskSeq<_>) =
+        checkNonNull (nameof source) source
+
+        if count < 0 then
+            raiseCannotBeNegative (nameof count)
+
+        match skipOrTake with
+        | Skip ->
+            // don't create a new sequence if count = 0
+            if count = 0 then
+                source
+            else
+                taskSeq {
+                    use e = source.GetAsyncEnumerator CancellationToken.None
+
+                    for _ in 1..count do
+                        let! step = e.MoveNextAsync()
```

Comment:
is more or ok not a better name?

Reply:
Updated. I should probably go over the code in general and find some kind of standard naming for often-used variable types.